prompt2model/dataset_retriever/description_dataset_retriever.py

```diff
@@ -341,16 +420,38 @@ def retrieve_top_datasets(
     def retrieve_dataset_dict(
         self,
         prompt_spec: PromptSpec,
+        data_transform: bool = False,
+        num_transform: int = 3000,
```

Comment:
Same point about the variable name `num_transform` being insufficiently descriptive as above